### PR TITLE
feat: story share

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/elements/src/share/ShareStory.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import styled from "styled-components";
+import { ShareEmailIcon, LinkIcon, ShareTwitterIcon } from "../icons";
+import { colors } from "../colors";
+import { fonts } from "../text";
+
+const ShareWrapper = styled.div`
+  width: 300px;
```
Comment (tobek) [inline packages/elements/src/share/ShareStory.tsx]: Maybe an edge case but this should probably have max width 100% - on smallest mobile viewport 320px, with page padding there might be less than 300px for this guy and it'd overflow.

Comment (sruddy) [reply packages/elements/src/share/ShareStory.tsx]: 👍 I meant to add a media query to this

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.31 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**